Sources_v3/StreamChatUI/ChatChannel/MessageComposerCommandCollectionViewCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class MessageComposerCommandCellView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+    // MARK: Properties
+
+    open var cellInfo: (title: String, subtitle: String, commandImage: UIImage?)? {
```

Comment:
Please rename to something else, like `content` i.e.

Reply:
Done, thanks, that sounds probably a lot better :)